app/javascript/components/auth-key-pair-cloud/auth-key-pair-cloud-form.schema.js

```diff
@@ -0,0 +1,48 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+import { API } from '../../http_api';
+
+const providerOptions = () => API.get(`/auth_key_pair_cloud/ems_form_choices`).then(data => [
+  ...data.ems_choices.map(({ id, name }) => ({ value: id, label: name })),
+
+]);
+
+function createSchema() {
+  const fields = [
+    {
+      id: 'name',
+      name: 'name',
+      label: __('Name'),
+      component: componentTypes.TEXT_FIELD,
+      validateOnMount: true,
+      validate: [{
+        type: validatorTypes.REQUIRED,
+      }, {
+        type: validatorTypes.PATTERN,
+        message: __('Please provide a valid git URL'),
+      }],
```

Comment:
You need an actual pattern for this to work, see the example [here](https://data-driven-forms.org/schema/pattern-validator#heading-patternvalidator) by clicking on the `<>` button